stripe/res/values/styles.xml

```diff
@@ -43,6 +43,13 @@
     <style name="Stripe.CardInputWidget.TextInputLayout"
         parent="Stripe.Base.CardInputWidget.TextInputLayout" />
 
+    <style name="Stripe.Base.CardMultilineWidget.TextInputLayout"
+        parent="Widget.Design.TextInputLayout">
+    </style>
+
+    <style name="Stripe.CardMultilineWidget.TextInputLayout"
+        parent="Stripe.Base.CardInputWidget.TextInputLayout" />
```

Comment:
Should this be `Stripe.Base.CardMultilineWidget.TextInputLayout`?

Reply:
whoops, good catch.